src/main/java/frc/robot/RobotContainer.java

```diff
@@ -24,17 +24,18 @@
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
   private final SwerveSubsystem swerveSubsystem = new SwerveSubsystem();
+  private final IntakeSubsystem intakeSubsystem = new IntakeSubsystem();
+
 
   private final AutonSubsytem autonSubsytem = new AutonSubsytem(swerveSubsystem);
   private SimulationSubsystem simulationSubsystem;
   
   private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
-  private final ExampleCommand m_autoCommand = new ExampleCommand(m_exampleSubsystem);
 
   private static BatterySubsystem batterySubsystem;
 
   /** The container for the robot. Contains subsystems, OI devices, and commands. */
-  public RobotContainer() {
+  private RobotContainer() {
```

Comment:
Should be public

Reply:
@Tiny-AKA-Chance @Carternix This should be public.  

All Constructors need to be public or else you can't create the class.  

Let's not close conversations without changing code unless you put more information in the conversation about why no changes were needed.